Content.Shared/Chemistry/EntitySystems/SolutionContainerSystem.cs

```diff
@@ -0,0 +1,341 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using Content.Shared.Chemistry.Components;
+using Content.Shared.Chemistry.Components.SolutionManager;
+using Content.Shared.Chemistry.Reaction;
+using Content.Shared.Chemistry.Reagent;
+using Content.Shared.Examine;
+using JetBrains.Annotations;
+using Robust.Shared.GameObjects;
+using Robust.Shared.IoC;
+using Robust.Shared.Localization;
+using Robust.Shared.Log;
+using Robust.Shared.Maths;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
+using Robust.Shared.Utility;
+
+namespace Content.Shared.Chemistry.EntitySystems
+{
+    /// <summary>
+    /// This event alerts system that the solution was changed
+    /// </summary>
+    public class SolutionChangedEvent : EntityEventArgs
+    {
+        public EntityUid Owner { get; }
```

Comment:
Not sure if this was just moved but none of the subscribers even need this property as they're all directed events?

Reply:
Removed